my-next-app/components/SignUpForm.tsx

```diff
@@ -198,7 +138,7 @@ const SignUpForm = () => {
                 }}
               />
             </div>
-            {errors.passwordConfirm && <p className="mt-2 text-sm text-red-600">{errors.passwordConfirm}</p>}
+            {errors.passwordConfirm && <p className="mt-2 text-sm text-red-600">{(errors.passwordConfirm as any).message}</p>}
```

Comment:
```suggestion
            {errors.passwordConfirm && <p className="mt-2 text-sm text-red-600">{errors?.passwordConfirm?.message}</p>}
```
so you avoid TS complain